client/src/store/modules/user.js

```diff
@@ -3,7 +3,10 @@
  * =============================================================
  */
 const initialState = {
-  user: __PRELOADED_STATE__.user
+  user: __PRELOADED_STATE__.currentUser,
```

Comment:
Changing this to currentUser is going to break other code in the app. Is name change necessary?

Reply:
This is resolved.